Mathlib/MeasureTheory/Constructions/BorelSpace/Metric.lean

```diff
@@ -237,7 +237,7 @@ section NormedAddCommGroup
 
 variable [MeasurableSpace α] [NormedAddCommGroup α] [OpensMeasurableSpace α] [MeasurableSpace β]
 
-@[measurability]
+@[fun_prop, measurability]
```

Comment:
I am not sure if `continuous_norm` should be tagged `fun_prop` instead.

Reply:
Not documented yet, I'm working on documentation. Right now, keep on pinging me to I can see what kind of questions come up and I can include them in the doc. 

Yes it can be inferred from continuity but providing specialized measurability theorem will make proofs about norm faster as `fun_prop` does not have to do as many steps.

Reply:
Thanks for the quick answer!